src/org/thoughtcrime/securesms/connect/DcHelper.java

```diff
@@ -247,6 +251,8 @@ public static void setStockTranslations(Context context) {
     dcContext.setStockTranslation(123, context.getString(R.string.aeap_explanation));
     dcContext.setStockTranslation(162, context.getString(R.string.multidevice_qr_subtitle));
     dcContext.setStockTranslation(163, context.getString(R.string.multidevice_transfer_done_devicemsg));
+    dcContext.setStockTranslation(170, context.getString(R.string.chat_protection_enabled) + " " + R.string.tap_learn_more);
+    dcContext.setStockTranslation(171, context.getString(R.string.chat_protection_broken) + " " + R.string.tap_learn_more);
```

Comment:
probably okayish in most languages, we do not add full stop or so here, it could also be two views. in case issues pop up, we can add the two additional strings.

Reply:
otoh, thinking it over, the "hack" would be required on all, so, maybe better add the two additinal strings and keep the setTranslation() code clean and straight-forward